frame/balances/src/lib.rs

```diff
@@ -513,7 +513,11 @@ pub mod pallet {
 	pub type TotalIssuance<T: Config<I>, I: 'static = ()> = StorageValue<_, T::Balance, ValueQuery>;
 
 	/// The balance of an account.
-	///
+	/// ```
+	///  impl Balance for Runtime {
+	///    type AccountStore = StorageMapShim<Self::Account<Runtime>, frame_system::Provider<Runtime>, AccountId, Self::AccountData<Balance>>
```

Comment:
Some docs on what this does would be nice.

Reply:
@ECJ222 this is still not done.

Reply:
Please could you explain a little bit more on this, can I put the explanation above the updated code?

Reply:
sorry but doc still is not really sensible. You should explain the context, what options are available (store the data in this pallet or in system), which one is better, what tradeoff exists etc. 

I really don't see any value in adding this one example block.

Reply:
I made an update to my initial pr @kianenigma